### PR TITLE
OKTA-421006 Identity Engine Post Tuscany - Combine run sample pages and move up level

### PR DESCRIPTION
**Pull request by brianduffield-okta**
## Description:
- **What's changed?** Removed the following pages from the current TOC:

https://developer.okta.com//docs/guides/oie-embedded-sdk-overview/main/
https://developer.okta.com/docs/guides/oie-embedded-sdk-run-sample/android/main/
https://developer.okta.com/docs/guides/oie-embedded-widget-overview/main/
https://developer.okta.com/docs/guides/oie-embedded-widget-run-sample/aspnet/main/

And combined the content into a top-level section title: Run the sample apps

/docs/guides/oie-embedded-common-run-samples/android/main/

> Note: We may need a second look at the left-hand nav as I believe other tickets were touching layout.

> Note 2: I did not include re-directs in this PR; will address this in a separate PR next wee.

- **Is this PR related to a Monolith release?** n/a

### Resolves:

* [OKTA-421006](https://oktainc.atlassian.net/browse/OKTA-421006)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-common-run-samples/index.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+title: Run the sample apps
+excerpt: Steps to run the sample widget and SDK applications.
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-common-run-samples/index.md:3]:
```suggestion
excerpt: Steps to run the sample Widget and SDK applications.
```

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-common-run-samples/main/index.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+excerpt: Run the sample apps
+layout: Guides
+---
+
+<ApiLifecycle access="ie" /><br>
+<ApiLifecycle access="Limited GA" /><br>
+
+After you have successfully created and set up your Okta org and downloaded and set up the SDK, you can run both the sample widget app and the sample SDK app.
+
+## The widget flow
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-common-run-samples/main/index.md:12]:
```suggestion
## The embedded Widget app flow
```

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-common-run-samples/main/index.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+In tandem with the embedded SDK, the widget integrates into your app by using a native language interface and communicates directly with the Okta APIs. The following summarizes the flow.
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-common-run-samples/main/index.md:14]:
```suggestion
In tandem with the embedded SDK, the Widget integrates into your app by using a native language interface and communicates directly with the Okta APIs. The following diagram shows the flow.
```

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-common-run-samples/main/index.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
+<div class="common-image-format">
+
+![Displays the Embedded SDK widget flow](/img/oie-embedded-sdk/embedded-widget-overview.png)
+
+</div>
+
+## Run the widget sample app
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-common-run-samples/main/index.md:22]:
```suggestion
## Run the Widget sample app
```

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-common-run-samples/main/index.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+
+<StackSelector snippet="runwidgetapp" noSelector />
+
+### Start your work with the use cases
+
+After you have successfully run the sample app, the next step is to build your integration by using the sample app as your guide. See [Start with a use case overview](/docs/guides/oie-embedded-widget-use-cases/aspnet/oie-embedded-widget-use-case-overview/) to explore the available use cases with the Widget.
+
+## The SDK flow
+
+The SDK integrates into your app by using a native language interface and communicates directly with Okta APIs by using a REST interface. The following is a summary of the SDK flow.
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-common-run-samples/main/index.md:34]:
```suggestion
The SDK integrates into your app by using a native language interface and communicates directly with Okta APIs by using a REST interface. The following diagram illustrates the SDK flow.
```

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-common-run-samples/main/index.md [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+
+<StackSelector class="cleaner-selector"/>
+
+<StackSelector snippet="runwidgetapp" noSelector />
+
+### Start your work with the use cases
+
+After you have successfully run the sample app, the next step is to build your integration by using the sample app as your guide. See [Start with a use case overview](/docs/guides/oie-embedded-widget-use-cases/aspnet/oie-embedded-widget-use-case-overview/) to explore the available use cases with the Widget.
+
+## The SDK flow
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-common-run-samples/main/index.md:32]:
```suggestion
## The embedded SDK app flow
```

**Review by vanngo-okta — APPROVED** [review]
Java part looks good to me! 🚢 
A few nits (ignore the "Widget" updates if the convention is to use lower "widget" - thanks)!

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-common-run-samples/main/index.md [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+
+After you have successfully created and set up your Okta org and downloaded and set up the SDK, you can run both the sample widget app and the sample SDK app.
+
+## The widget flow
+
+In tandem with the embedded SDK, the widget integrates into your app by using a native language interface and communicates directly with the Okta APIs. The following summarizes the flow.
+
+<div class="common-image-format">
+
+![Displays the Embedded SDK widget flow](/img/oie-embedded-sdk/embedded-widget-overview.png)
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-common-run-samples/main/index.md:18]:
```suggestion
![Displays the Embedded SDK Widget flow](/img/oie-embedded-sdk/embedded-widget-overview.png)
```

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-common-run-samples/main/ios/runwidgetapp.md [hunk 8]
```diff
@@ -0,0 +1,2 @@
+
+> **Note:** There is no widget sample application for iOS.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-common-run-samples/main/ios/runwidgetapp.md:2]:
```suggestion
> **Note:** There is no Widget sample application for iOS.
```

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-common-run-samples/main/android/runwidgetapp.md [hunk 9]
```diff
@@ -0,0 +1,2 @@
+
+> **Note:** There is no widget sample application for Android.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-common-run-samples/main/android/runwidgetapp.md:2]:
```suggestion
> **Note:** There is no Widget sample application for Android.
```

**Review by susanharper-okta — APPROVED** [review]
just Widget suggestions. Looks good to me besides that. Also built it locally.

**Pull request merged**